src/hooks/useFocusTrap.ts

```diff
@@ -0,0 +1,77 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
```

Comment:
Please add some tests

Reply:
I added the tests for focusTrap directly in InlineDialog.cy.tsx The whole section `Modality` test it.
Seems like none of the hooks have an independent test file 🤔 , we test them in use in the component.
You feel like it's not enough?

Reply:
`src/hooks/useFocusTrap.ts`, the hook isn't component specific.

Reply:
Will add.